rust_src/src/multibyte.rs

```diff
@@ -92,11 +92,14 @@ impl LispStringRef {
         s.size_byte >= 0
     }
 
+    // Same as SDATA function
     pub fn data_ptr(&mut self) -> *mut c_uchar {
         let s = unsafe { self.u.s };
         s.data as *mut c_uchar
     }
 
+    // Same as SSDATA function
```

Comment:
Previously I used `.const_data_ptr() as *const i8` for `SSDATA`.

Reply:
I think I also didn't use this method in some PR, so I added the comments.